.happy/terraform/modules/batch/main.tf

```diff
@@ -10,7 +10,7 @@ resource aws_batch_job_definition batch_job_def {
 {
   "jobRoleArn": "${var.batch_role_arn}",
   "image": "${var.image}",
-  "memory": 28000,
+  "memory": "${var.batch_container_memory_limit}",
```

Comment:
should this be an int or a string? can we remove the quotes?

Reply:
yes. quotes needs to be removed. will test on rdev and update the PR.